src/object/srv_obj_migrate.c

```diff
@@ -187,8 +187,11 @@ obj_tree_create(daos_handle_t toh, void *key, size_t key_size,
 
 	*rootp = tmp_root;
 out:
-	if (rc < 0)
-		dbtree_delete(toh, BTR_PROBE_EQ, &key_iov, NULL);
+	if (rc < 0) {
+		rc2 = dbtree_delete(toh, BTR_PROBE_EQ, &key_iov, NULL);
+		if (rc2)
+			D_WARN("failed to delete \"%s\": "DF_RC"\n", (char *)key, DP_RC(rc2));
```

Comment:
I made a similar comment on master version of the PR.  This may not be a character string...A key can be anything and doesn't need to be NULL terminated.  DF_KEY and DP_KEY will work here.

Reply:
Thank you very much!

D_WARN("failed to delete "DF_KEY": "DF_RC"\n", DP_KEY(&key_iov), DP_RC(rc2));

Is it correct now?